addons/web/static/tests/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -227,6 +228,49 @@ QUnit.module("Fields", (hooks) => {
         );
     });
 
+    QUnit.test(
+        "widget many2many_tags_avatar list view - don't crash on keyboard navigation and delete buttons",
+        async function (assert) {
+            // Assign the 3 partner records as partner_ids of the first turtle.
+            serverData.models.turtle.records[0].partner_ids = [1, 2, 4];
+
+            await makeView({
+                type: "list",
+                resModel: "turtle",
+                serverData,
+                arch: /*xml*/ `
+                    <tree editable="bottom">
+                        <field name="partner_ids" widget="many2many_tags_avatar"/>
+                    </tree>
+                `,
+            });
+
+            // Select the 2nd row and enter edit mode on the x2many field.
+            await click(target, ".o_data_row:nth-child(2) .o_list_record_selector input");
+            await click(target, ".o_data_row:nth-child(2) .o_data_cell");
+
+            // Pressing left arrow should focus on the right-most (second) tag.
+            await triggerHotkey("arrowleft");
+            assert.strictEqual(
+                target.querySelector(".o_data_row:nth-child(2) .o_field_tags span:nth-child(2)"),
+                document.activeElement
+            );
+
+            // Pressing left arrow again should not crash and should focus on the first tag.
+            await triggerHotkey("arrowleft");
+            assert.strictEqual(
+                target.querySelector(".o_data_row:nth-child(2) .o_field_tags span:nth-child(1)"),
+                document.activeElement
+            );
+
+            // And oh, there should be delete buttons on the tags since we are in edit mode.
+            assert.containsN(target, ".o_field_tags span .o_delete", 2);
```

Comment:
maybe this assert could be moved to the previous test, which is more basic, and we can leave this one with keynav stuff. What do you think?

Reply:
That's true. And the new test will only for navigation, right?

Reply:
yep